### PR TITLE
FileSpec from file

### PR DESCRIPTION
**Pull request by yahavi**

### CODE REVIEW

File: ArtifactoryGenericUpload/task.json [hunk 1]
```diff
@@ -27,18 +27,37 @@
             "required": true,
             "helpMarkDown": "Artifactory service used for uploading artifacts."
         },
+        {
+            "name": "specSource",
+            "type": "radio",
+            "label": "Upload spec source",
```
Comment (eyalbe4) [inline ArtifactoryGenericUpload/task.json]: I suggest changing "Upload spec source" to "File spec source"

File: ArtifactoryGenericDownload/task.json [hunk 2]
```diff
@@ -27,18 +27,37 @@
             "required": true,
             "helpMarkDown": "Artifactory service used for downloading artifacts."
         },
+        {
+            "name": "specSource",
+            "type": "radio",
+            "label": "Download spec source",
```
Comment (eyalbe4) [inline ArtifactoryGenericDownload/task.json]: I suggest changing "Download spec source" to "File spec source"

File: tests/testUtils.js [hunk 3]
```diff
@@ -123,7 +123,7 @@ function setVariables(variables) {
 }
 
 function setInputs(inputs) {
```
Comment (eyalbe4) [inline tests/testUtils.js]: I suggest adding a description comment for this function.

**Comment by eyalbe4** [comment]
Added my inline comments.
Looks great to me :)

File: ArtifactoryGenericUpload/task.json (hunk 1 above)
Comment (yahavi) [reply ArtifactoryGenericUpload/task.json]: Fixed

File: ArtifactoryGenericDownload/task.json (hunk 2 above)
Comment (yahavi) [reply ArtifactoryGenericDownload/task.json]: Fixed

File: tests/testUtils.js (hunk 3 above)
Comment (yahavi) [reply tests/testUtils.js]: Fixed and changed setInputs to mockGetInputs.

**Pull request merged**